include/SFML/Audio/SoundStream.hpp

```diff
@@ -327,11 +327,8 @@ class SFML_AUDIO_API SoundStream : public SoundSource
     ////////////////////////////////////////////////////////////
     void awaitStreamingThread();
 
-    enum
-    {
-        BufferCount   = 3, //!< Number of audio buffers used by the streaming loop
-        BufferRetries = 2  //!< Number of retries (excluding initial try) for onGetData()
-    };
+    static constexpr unsigned int BufferCount{3U};
+    static constexpr unsigned int BufferRetries{2U};
```

Comment:
```suggestion
    static constexpr unsigned int BufferCount{3};
    static constexpr unsigned int BufferRetries{2};
```
There's no need to add that suffix

Reply:
It will if you crank up the warning level; but OK.

Reply:
If we crank up the warning level? The warnings are already cranked to the max.

https://github.com/SFML/SFML/blob/master/cmake/CompilerWarnings.cmake

Reply:
Interesting; I've had to add suffixes in the past to prevent signed vs unsigned mismatch warnings.

Reply:
It's needed in some places, but since you're initializing an unsigned int, the compiler can infer